content/en/docs/languages/go/_index.md

```diff
@@ -26,7 +26,7 @@ spelling: cSpell:ignore Isberner Klerk Malte youtube
   by [Jean de Klerk](https://github.com/jadekler), Google. October 8, 2020.
 - **Talking to Go gRPC Services Via HTTP/1**
   <a class="icon" href="https://youtu.be/Vbw8h0RCn2E"><i class="fab fa-youtube"></i></a>
-  <a class="icon" href="https://static.sched.com/hosted_files/grpcconf20/c9/TalkingToGoGRPCviaHTTP1-gRPCConf2020-MalteIsberner.pdf"><i class="far fa-file"></i></a><br>
+  <a class="icon" href="https://static.sched.com/hosted_files/grpcconf20/c9/TalkingToGoGRPCviaHTTP1-gRPCConf2020-MalteIsberner.pdf"><i class="fas fa-file"></i></a><br>
```

Comment:
AFAIR, the class `icon` on the anchor element would give us a bit of space between consecutive icons. Right now the icons are stuck together, e.g.:

> ![image](https://user-images.githubusercontent.com/4140793/102674390-d53c5200-4163-11eb-854f-98e46b7faee4.png)

Can you restore the space?

Reply:
Oh strange - it looks fine in development on every browser...but the spacing is squashed on Netlify... I'll add an `l-icon` class.

Reply:
Updated!